tests/e2e/TimeoutConstants.ts

```diff
@@ -97,6 +96,18 @@ export const TimeoutConstants = {
      */
     TS_IMPORT_PROJECT_DEFAULT_POLLING: Number(process.env.TS_SELENIUM_DEFAULT_POLLING) || 50_000,
 
+    // ---------------------------------------- OPEN EDITORS ------------------------------------------
```

Comment:
New constants look like the ones specific to "**Editor**" section below.
Do we really need dedicated section for new constants?

Reply:
okay